articles/clients/client-settings/native/index.md

```diff
@@ -36,8 +32,6 @@ By default, the [settings](${manage_url}/#/clients/${account.clientId}/settings)
 
 - **Client Type**: The type of client you are implementing. Depending on which you choose, the available settings differ to show you only the settings applicable to your Client Type. You can change this value at any time by selecting one of the following: Native, Non Interactive Client, Regular Web Application, or Single Page Application.
```

Comment:
You can update this description for each specific doc. Keep the first sentence and add sth like "for mobile apps the type should be native"